tests/sdk_test.cpp

```diff
@@ -20,9 +20,93 @@
  */
 
 #include "sdk_test.h"
+#include "mega/testhooks.h"
```

Comment:
jenkins reports:
```
tests/sdk_test.cpp:23:28: fatal error: mega/testhooks.h: No such file or directory
 #include "mega/testhooks.h"
                            ^
compilation terminated.
```

Reply:
thanks very much, I should have checked the sdk_tests.   Updated now